src/views/schemaBrowser.ts

```diff
@@ -103,7 +104,197 @@ export default class schemaBrowser {
+                  title: `Renaming ${object.name}...`
+                }, async () => {
+                  await Schemas.renameObject(object.schema, object.name, name, object.type);
+                });
+                
+                vscode.window.showInformationMessage(`Renamed ${object.name} to ${name}`);
+                this.cache = {};
+                this.refresh();
+              } catch (e) {
+                vscode.window.showErrorMessage(e.message);
+              }
+            }
+          } else {
+            vscode.window.showErrorMessage(`Name cannot be blank`);
+          }
+        }
+      }),
+      
+      vscode.commands.registerCommand(`vscode-db2i.clearData`, async (object: SQLObject) => {
+        if (object) {
+          const result = await vscode.window.showWarningMessage(`Are you sure you want to clear ${object.name}?`, {
+            modal: true,
+          }, 'No', 'Yes');
+
+          if(result === 'Yes') {
+            try {
+              await vscode.window.withProgress({
+                location: vscode.ProgressLocation.Notification,
+                title: `Clearing ${object.name}...`
+              }, async () => {
+                await Table.clearFile(object.system.schema, object.system.name);
+              });
+
+              vscode.window.showInformationMessage(`${object.name} cleared`);
+            } catch (e) {
+              vscode.window.showErrorMessage(e.message);
+            }
+          }
+        }
+      }),
+
+      vscode.commands.registerCommand(`vscode-db2i.copyData`, async (object: SQLObject) => {
+        if (object) {
+          const base = loadBase();
+          const page = await base.customUI()
+            .addInput('toFile', 'To File')
+            .addInput('toLib', 'Library')
+            .addInput('fromMbr', 'From member', 'Name, generic*, *FIRST, *ALL', {
+              default: '*FIRST'
+            })
+            .addInput('toMbr', 'To member or label', 'Name, *FIRST, *FROMMBR, *ALL', {
+              default: '*FIRST'
+            })
+            .addSelect('mbrOpt', 'Replace or add records', [
+              {text: '*NONE', description: '*NONE', value: '*NONE'},
+              {text: '*ADD', description: '*ADD', value: '*ADD'},
+              {text: '*REPLACE', description: '*REPLACE', value: '*REPLACE'},
+              {text: '*UPDADD', description: '*UPDADD', value: '*UPDADD'},
+            ])
+            .addSelect('crtFile', 'Create file', [
+              {text: '*NO', description: '*NO', value: '*NO'},
+              {text: '*YES', description: '*YES', value: '*YES'},
+            ])
+            .addSelect('outFmt', 'Print format', [
+              {text: '*CHAR', description: '*CHAR', value: '*CHAR'},
+              {text: '*HEX', description: '*HEX', value: '*HEX'},
+            ])
+            .addButtons(
+              {id: 'copy', label:'Copy'},
+              {id: 'cancel', label:'Cancel'}
+            )
+            .loadPage<any>('Copy File');
```

Comment:
Include the qualified object name in the page title.
```suggestion
            .loadPage<any>(`Copy File - ${object.schema}.${object.name}`);
```